blueprinter.gemspec

```
@@ -24,7 +24,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency "oj", "~> 3.13"
   s.add_development_dependency "yajl-ruby", "~> 1.4"
   s.add_development_dependency "pry", "~> 0.14"
-  s.add_development_dependency "activerecord", "~> 7.0"
+  s.add_development_dependency "activerecord", "> 5.1.2"
```

This gets tests to work in ruby 2.6.9, I could probably go with a newer version than 5.1.2, I chose that just because that is what the version was before your updates in "initial commit" @ritikesh

---

```suggestion
  s.add_development_dependency "activerecord", ">= 5.2"
```
let's go with latest 5.x series at least.